src/__tests__/lib/exampleData.js

```diff
@@ -244,6 +272,56 @@ export const streamMessage = (extra?: $Rest<Message, {}>): Message => {
   return deepFreeze({ ...baseMessage, ...extra });
 };
 
+/* ========================================================================
+ * Outbox messages
+ *
+ * (Only stream messages for now. Feel free to add PMs, if you need them.)
+ */
+
+/** An outbox message with no interesting data. */
+const outboxMessageBase: $Diff<Outbox, {| id: mixed, timestamp: mixed |}> = deepFreeze({
+  isOutbox: true,
+  isSent: false,
+
+  avatar_url: 'https://example.com/avatar.png',
+  content: '<p>Test.</p>',
+  display_recipient: 'test',
+  // id: ...,
+  markdownContent: 'Test.',
+  narrow: [{ operator: 'stream', operand: 'test' }],
+  reactions: [],
+
```

Comment:
does this blank line signify something? I'm missing it. (The other one does, and is helpful.)